src/controllers/controller.bubble.js

```diff
@@ -122,16 +138,13 @@ module.exports = DatasetController.extend({
 	/**
 	 * @private
 	 */
-	_resolveElementOptions: function(point, index) {
+	_resolveDataElementOptions: function(point, index) {
```

Comment:
why add `Data` to the name? I liked the name `_resolveElementOptions`

Reply:
Now there are two resolving functions in the base controller: `_resolveDatasetElementOptions` and `_resolveDataElementOptions`. The former resolves dataset element options (i.e. line element options) and the latter resolves data element options (arc, point and rectangle element options). I wanted to make it clear that each method resolves which element options.

Reply:
Since some controllers have custom resolving logic and extend resolving methods, I think it is better to have these methods separately.

Reply:
Ah, the names `_resolveDataElementOptions` and `_resolveDatasetElementOptions` are so similar I did not notice there were two different methods

Reply:
(I edited @nagix's comment, changed 2nd `_datasetElementOptions` to `_dataElementOptions`)

Reply:
Thanks @kurkle